packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -916,6 +925,18 @@ export const addEdge = async (id: string, edge: WorkflowEdge) => {
 	return response.data ?? null;
 };
 
+export const addOrUpdateAnnotation = async (id: string, annotation: WorkflowAnnotation) => {
+	console.log('>> workflowService.addOrUpdateAnnotation');
+	const response = await API.post(`/workflows/${id}/annotation`, annotation);
+	return response.data ?? null;
+};
+
+export const removeAnnotation = async (id: string, annotationId: string) => {
+	console.log('>> workflowService.removeAnnotation');
```

Comment:
If intention feel free to just resolve this comment I do not mind past you being aware